static/app/components/charts/eventsChart.spec.jsx

```diff
@@ -43,135 +42,26 @@ describe('EventsChart', function () {
         router={router}
         {...props}
       />,
-      routerContext
+      {context: routerContext}
     );
   }
 
-  beforeEach(function () {
-    globalSelection.updateDateTime.mockClear();
-    mockZoomRange(1543449600000, 1543708800000);
-    wrapper = renderChart();
-
-    // XXX: Note we spy on this AFTER it has already rendered once!
-    render = jest.spyOn(wrapper.find('ChartZoom').instance(), 'render');
-  });
-
-  it('renders', function () {
-    expect(render).toHaveBeenCalledTimes(0);
-  });
-
-  it('re-renders if period from props changes', function () {
-    wrapper.setProps({period: '7d'});
-    wrapper.update();
-    expect(render).toHaveBeenCalledTimes(1);
-  });
-
-  it('re-renders if query from props changes', function () {
-    wrapper.setProps({query: 'newQuery'});
```

Comment:
is this being reflected in the UI? if we can't target the DOM element, can we take a screenshot?

Reply:
echarts is entirely mocked out in all our tests

Reply:
maybe we can keep this test some other way, it must make a request or something